packages/components/src/components/swipeable-notification/swipeable-notification.scss

```diff
@@ -0,0 +1,52 @@
+.swipeable-notification {
+    width: 100%;
+    min-height: 4.8rem;
+    border-radius: $BORDER_RADIUS;
+    padding: 0.8rem;
+    background-color: $color-grey-5;
+    display: flex;
+    justify-content: space-between;
+    text-decoration: none;
+
+    &--failure {
+        background-color: $color-red-11;
+    }
+    &--success {
+        background-color: $color-green-8;
+    }
+    &-content {
+        align-self: center;
+    }
+    &-time {
+        align-self: flex-end;
+    }
+    &-appear,
+    &-enter {
+        transform: translateX(-100vw);
```

Comment:
Question: isn't it possible to use percentage in these animations?

Reply:
@akmal-deriv unfortunately, not. the parent can have a width that is less than viewed-width. That's why viewed-width is used here :)